### ISSUE TITLE
Plugin error during initial pass stops watch

### ISSUE THREAD

**Issue by gordonmleigh** [rollup/rollup#3215]
- Rollup Version: 1.26.3
- Operating System (or Browser): OS X 10.14
- Node Version: 12.11.1

#### Files

**index.js:**

```js
func tion foo() {} // note space
```

**rollup.config.js:**

```js
import resolve from 'rollup-plugin-node-resolve';
import commonjs from 'rollup-plugin-commonjs';

export default {
  input: './index.js',
  output: [
    {
      file: 'lib/bundle.js',
      format: 'cjs',
      sourcemap: true,
    },
  ],

  plugins: [resolve(), commonjs()],
};
```

#### Steps

1. Run `rollup -c -w`
2. Fix syntax error in js file and save

### Expected Behaviour

The build should fail, but then run again when the file is fixed.

### Actual Behavior

Error is shown on first build:

```
rollup v1.26.3
bundles ./index.js → lib/bundle.js...
[!] (plugin commonjs) SyntaxError: Unexpected token (1:4) in /../rollup-watch-repro/index.js
index.js (1:4)
1: fun ction foo() {}
       ^
SyntaxError: Unexpected token (1:4) in /../rollup-watch-repro/index.js
```

After save, nothing happens. **Note that this problem only happens if there is a _plugin_ error** (if the commonjs plugin is removed, then the file is rebundled as expected).

### Related issues

Following issues have no repros but are related: #3172, #2534

**Comment by petterek** [comment]
In my case where I have multiple entry points / files this also happens if I have some syntax error in the ts files after while "watching"

**Comment by lukastaegert** [comment]
Thanks for spotting the relation to plugin errors, fix upcoming.

**Comment by lukastaegert** [comment]
Fix at #3219
